<!DOCTYPE HTML>
<!--
	Massively by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Publications</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<a href="index.html" class="logo">Top Of Page</a>
					</header>

				<!-- Nav -->
					<nav id="nav">
						<ul class="links">
							<li><a href="index.html">Home</a></li>
							<li><a href="cv.html">CV</a></li>
							<li class="active"><a href="publications.html">Publications</a></li>
						</ul>
						<ul class="icons">
							<li><a href="#" class="icon brands fa-github"><span class="label">GitHub</span></a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">

						<!-- Post -->
							<section class="post">
								<p>Devlin, S., Treloar, T., Creagar, M., & Cassels, S. (2020). “An iterative Markov rating method”, 
									<em>Journal of Quantitative Analysis in Sports</em> (published online ahead of print 2020), 000010151520190070. 
									doi: <a href="https://doi.org/10.1515/jqas-2019-0070">https://doi.org/10.1515/jqas-2019-0070</a>. </p>
							</section>
							<section class="post"
								 <p>Creagar, M., Kamp, T., Fore, R., Friend, D., Disselkoen, C., Stanislaw, L., Wang, S., Alfieri, J., 
								Henry, C., DeJongh, M., Best, A.,Lindsey, W., and Tintle, N. (2018). "Developing and evaluating integrated metabolic regulatory models for microbial life",
								<em>Under Revision</em>.</p> <p>Preliminary code: <a href="http://nathantintle.com/supplemental/iMRM_Preliminary_Code.zip">http://nathantintle.com/supplemental/iMRM_Preliminary_Code.zip</a></p>
								<p>Method description: <a href="http://nathantintle.com/supplemental/iMRM_method_description.pdf">http://nathantintle.com/supplemental/iMRM_method_description.pdf</a></p>	
				<p font-size:12px><em>Abstract:</em> A number of papers have described methods to include transcriptional regulatory networks (TRNs) in the development of metabolic models. However, in general, these models do not allow for statistical 
					uncertainty in TRN interactions. Furthermore, methods which do allow for uncertainty do so in a non-rigorous manner which, effectively, downweights prior transcriptomics data to the point where it has little impact on the resulting model. 
					To combat these limitations, we developed an integrated metabolic regulatory model (iMRM). This novel approach explicitly models the statistical uncertainty in gene state activity inferences by using gene activity state estimates, and the iMRM 
					process measures the flux through reactions for a sole single-cell organism. This allows for benefits such as differential thresholds for gene activity for different genes, a stronger correlation with experimental flux data, and the application 
					of a non-Boolean confidence metric. Furthermore, this is the first such metabolic model to be grounded on a statistically rigorous foundation.</p>
				<p font-size:12px> This work was made possible by the National Science Foundation grant MCB-1715211.</p>

							</section>
							<section class="post">
								<p>Veenstra, J., Creagar, M., Frantz, R., Sorensen, E., Westra, J., and Tintle, N. (2018). "Methods for assessing and correcting genotypic uncertainty in set-based analysis of medium
								to large sets of genetic data", <em>Under Revision</em>.</p>
								<p font-size:12px><em>Abstract:</em> The impact of genotypic uncertainty (e.g., genotype errors) in sequence data on gene-based tests of phenotype association (e.g., variance-components and burden tests) 
									are fairly well understood with genotype errors that are not associated with the phenotype of interest reducing power and genotype errors that are associated with the phenotype 
									increasing the type I error rate. However, little work has been done to establish how these patterns manifest themselves in large ‘biobank’ style datasets, where data has been 
									aggregated from numerous sources each with potentially different patterns of genotype uncertainty. In this manuscript we use simulated data to demonstrate the aggregate effects
									of genotype uncertainty on downstream statistical analyses in biobank-style datasets. We then propose two methods to adjust downstream data processing to mitigate the impact of 
									these errors in order to appropriately control the type I error rate while maximizing statistical power.</p>
								<p font-size:12px>This work was made possible by the National Institute of Health grant NIH-NHGRI-2R15HG006915.</p>
							</section>

					</div>

				<!-- Footer -->
					<footer id="footer">
						<section class="split contact">
							<section>
								<h3>Email</h3>
								<p><a href="#">mcreagar2@huskers.unl.edu</a></p>
							</section>
						</section>
					</footer>

				<!-- Copyright -->
					<div id="copyright">
						<ul><li>&copy; Molly Creagar</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li></ul>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
